src/style.css

```diff
@@ -1 +1,7 @@
-/* styles go here */
+.form fieldset {
```

Comment:
According to the [Checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) you should use classes rather than tags for css styling.